sdk/attestation/azure-security-attestation/azure/security/attestation/_configuration.py

```diff
@@ -4,13 +4,9 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------
 
-from typing import NoReturn, TYPE_CHECKING
+from typing import TYPE_CHECKING
 
 from azure.core.configuration import Configuration
-from azure.core.pipeline import policies
-
-from ._version import VERSION
-from ._models import TokenValidationOptions
 
 if TYPE_CHECKING:
     # pylint: disable=unused-import,ungrouped-imports
```

Comment:
This suppression can be removed if the TokenCredential import is removed (since it's no longer used)
```suggestion
    # pylint: disable=ungrouped-imports
```